server/src/main/java/org/apache/druid/server/SegmentManager.java

```diff
@@ -219,9 +220,9 @@ private TableDataSource getTableDataSource(DataSourceAnalysis analysis)
    *
    * @throws SegmentLoadingException if the segment cannot be loaded
    */
-  public boolean loadSegment(final DataSegment segment, boolean lazy) throws SegmentLoadingException
+  public boolean loadSegment(final DataSegment segment, boolean lazy, CommonCallback loadFailed) throws SegmentLoadingException
```

Comment:
add a new `@param` in the method comment

Reply:
done